src/main/java/teammates/ui/newcontroller/WebApiServlet.java

```diff
@@ -44,7 +48,51 @@ protected void doDelete(HttpServletRequest req, HttpServletResponse resp) throws
 
     @SuppressWarnings("PMD.AvoidCatchingThrowable") // used as fallback
     private void invokeServlet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
-        // TODO
+        resp.setHeader("Strict-Transport-Security", "max-age=31536000");
```

Comment:
`<transport-guarantee>CONFIDENTIAL</transport-guarantee>` in `web.xml` won't do this job?

Reply:
I'm actually not sure what the best practice will be. `Strict-Transport-Security` header is a purist HTTP header approach (if anything I prefer using this header) while the `transport-guarantee` is more black-boxed. In fact, I'm not even sure they're doing the same thing.